src/BuildingRegistry.Api.BackOffice.Handlers.Lambda/Function.cs

```diff
@@ -82,9 +85,9 @@ protected override IServiceProvider ConfigureServices(IServiceCollection service
 
             builder
                 .RegisterModule(new DataDogModule(configuration))
-                .RegisterModule<EnvelopeModule>()
-                .RegisterModule(new EditModule(configuration, services, loggerFactory))
+                .RegisterModule(new CommandHandlingModule(configuration))
                 .RegisterModule(new BackOfficeModule(configuration, services, loggerFactory))
+                .RegisterModule(new EventHandlingModule(typeof(DomainAssemblyMarker).Assembly, eventSerializerSettings))
```

Comment:
I think event handling is registered through CommandHandlingModule -> AggregateSourceModule.